src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/CostsServiceTest.java

```diff
@@ -3,28 +3,35 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.test.util.ReflectionTestUtils;
 
 import java.util.Collections;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class CostsServiceTest {
-
-    @InjectMocks
-    private CostsService costService;
+    @Mock
+    private OverseasEntitiesService overseasEntitiesService;
 
     @BeforeEach
     void init() {
-        ReflectionTestUtils.setField(costService, "costAmount", "13.00");
```

Comment:
How about creating the costs service in the beforeeach?

Reply:
Went with suggestion